start

```diff
@@ -526,7 +526,7 @@ function upgrade_local() {
         "$NETWORK_ROOT_ACCOUNT_ID" \
         0 \
         "$NETWORK_PASSPHRASE" \
-        --json "$COREHOME/etc/config-settings/$LIMITS.json" \
+        --xdr <(stellar-xdr encode --type ConfigUpgradeSet --output single < "$COREHOME/etc/config-settings/$LIMITS.json") \
```

Comment:
The stellar-xdr cli doesn't have the encode command until the next release. So this PR is on hold until the next release.